docs/concepts/composition.md

```diff
@@ -1,156 +1,96 @@
-
-> Note that composite resources and compositions are _cluster scoped_ - they
-> exist outside of any Kubernetes namespace. A claim is a namespaced proxy for a
-> composite resource. This enables Crossplane to model complex relationships
-> between XRs that may span namespace boundaries - for example MySQLInstances
-> spread across multiple namespaces can all share a VPC that exists above any
-> namespace.
-
-## Creating A New Kind of Composite Resource
-
-New kinds of composite resource are defined by a platform builder. There are
-two steps to this process:
-
-1. Define your composite resource, and optionally the claim it offers.
-1. Specify one or more possible ways your composite resource may be composed.
-
-### Define your Composite Resource
-
-Composite resources are defined by a `CompositeResourceDefinition`:
+# Composite Resources
+
+## Overview
+
+Crossplane Composite Resources are opinionated Kubernetes Custom Resources that
+are _composed_ of [Managed Resources][managed-resources]. We often call them XRs
+for short.
+
+![Diagram of claims, XRs, and Managed Resources][xrs-and-mrs]
+
+> If you're already familiar with Composite Resources and looking for a detailed
+> configuration reference or some tips, tricks, and troubleshooting information,
+> try the [Composition Reference][xr-ref].
+
+Composite Resources are designed to let you build your own platform with your
+own opinionated concepts and APIs without needing to write a Kubernetes
+controller from scratch. Instead, you define the schema of your XR and teach
+Crossplane which Managed Resources it should compose (i.e. create) when someone
+creates the XR you defined.
+
+
+Below is an example of a Composite Resource:
+
+```yaml
+apiVersion: database.example.org/v1alpha1
+kind: XPostgreSQLInstance
+metadata:
+  name: my-db
+spec:
+  parameters:
+    storageGB: 20
+  compositionRef:
+    name: production
+  writeConnectionSecretToRef:
+    namespace: crossplane-system
+    name: my-db-connection-details
+```
+
+You define your own XRs, so they can be of whatever API version and kind you
+like, and contain whatever spec and status fields you need.
+
+## How It Works
+
+The first step towards using Composite Resources is configuring Crossplane so
+that it knows what XRs you'd like to exist, and what to do when someone creates
+one of those XRs. This is done using a `CompositeResourceDefinition` resource
+and one or more `Composition` resources.
+
+Once you've configured Crossplane with the details of your new XR you can either
+create one directly, or use a _claim_. Typically only the folks responsible for
+configuring Crossplane (often a platform or SRE team) have permission to create
+XRs directly. Everyone else manages XRs via a lightweight proxy resource called
+a Composite Resource Claim (or claim for short). More on that later.
+
+![Diagram combining all Composition concepts][how-it-works]
+
+> If you're coming from the Terraform world you can think of an XRD as similar
```

Comment:
Another note I would move. ;-)

Reply:
I think this one makes sense as a quote note. It's at the end of the heading so I don't feel it interrupts.